apps/examples/wifi_manager/wm_test/wm_test_stress_test2.c

```diff
@@ -500,3 +516,32 @@ void wm_run_stress_test2(struct wt_options *opt)
 	/*  Destroy queue */
 	wo_destroy_queue(g_wo_queue);
 }
+
+void wm_run_stress_test2(struct wt_options *opt)
+{
+	if (!opt->path) {
+		wm_run_stress_test2_single_ap(opt);
+	} else {
+		int ap_count = 0;
+		interop_ap_info_s *ap_info = NULL;
+		memset(&g_apconfig_list, 0, sizeof(g_apconfig_list));
+		int ret = 0;
+		ret = wifi_interop_read_file(&g_apconfig_list, opt->path);
+		if (ret < 0) {
+			printf("[WIO] -->%s: failed to open file \n", __FUNCTION__);
```

Comment:
Please use wm_test log(WT_LOG) not printf.

Reply:
Sir, changes done as suggested. Please review.